src/IECoreDelight/Renderer.cpp

```diff
@@ -1505,6 +1505,24 @@ class DelightRenderer final : public IECoreScenePreview::Renderer
 			const V2i &resolution = camera->getResolution();
 			screeenParameters.add( { "resolution", resolution.getValue(), NSITypeInteger, 2, 1, NSIParamIsArray } );
 
+			const int overscanLeft = static_cast<int>(camera->getOverscanLeft() * resolution.x);
```

Comment:
The Camera class also has a [`getOverscan()`](https://github.com/ImageEngine/cortex/blob/main/include/IECoreScene/Camera.h#L170) method that returns a boolean to say whether or not overscan should be active. We need to check that it is on before applying it.

Reply:
This should be fixed now. I added a check to see if getOverscan() is true or false and depending on that add the overscan parameters to the NSI screen.